pkgs/development/compilers/zig/0.10.nix

```diff
@@ -85,4 +86,6 @@ stdenv.mkDerivation rec {
     maintainers = with maintainers; [ aiotter andrewrk AndersonTorres ];
     platforms = platforms.unix;
   };
+} // {
+  pkgs = callPackage ../../../top-level/zig-packages.nix { };
```

Comment:
if we do end up doing this, this should be added to zig 0.9 as well

Reply:
So remove it or add it to 0.9?

Reply:
also something might need to be done for this to be spliced correctly

Reply:
> So remove it or add it to 0.9?

yep